src/SamplesApp/UITests.Shared/Windows_UI_Xaml_Controls/PasswordBoxTests/PasswordBoxPage.xaml.cs

```diff
@@ -0,0 +1,16 @@
+﻿using Uno.UI.Samples.Controls;
+using Windows.UI.Xaml.Controls;
+
+// The Blank Page item template is documented at https://go.microsoft.com/fwlink/?LinkId=234238
+
+namespace UITests.Shared.Windows_UI_Xaml_Controls.PasswordBoxTests
+{
+	[Sample("PasswordBox")]
```

Comment:
This, unfortunately, create a new top-level item in the sample app. We need to organize the sample app to reduce the number of top-level items.

Reply:
Currently `PasswordBox` related sample pages are in `TextBox` category, so you can move the sample there too 👍

Reply:
```suggestion
	[Sample("TextBox")]
```